srcpkgs/firebird3/files/firebird-fix-perm-fdb

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+RunUser=_firebird
+RunGroup=_firebird
+databasesconf='/etc/firebird/databases.conf'
+
+# !!! need shielding / !!!
+dir_sampledb='\/tmp'
```

Comment:
You can avoid shielding `/` by using a different separator character on line 16.